packages/fxa-auth-server/lib/senders/emails/partials/subscriptionUpdateBillingTry/index.mjml

```diff
@@ -5,7 +5,7 @@
 <mj-section>
   <mj-column>
     <mj-text css-class="text-body">
-      <span data-l10n-id="updateBilling">
+      <span data-l10n-id="subscriptionUpdateBillingTry">
```

Comment:
You may already know this, but changing the FTL string IDs can have some implications attached. I think it's fine here because Pontoon has strings in memory so if this is already been translated it shouldn't have to be retranslated, but if you ever need to change a string, we _have_ to update the ID with it.